testdata/storage_transformer/zarr.json

```diff
@@ -0,0 +1,40 @@
+{
+  "zarr_format": 3,
+  "node_type": "array",
+  "shape": [
+    1
+  ],
+  "data_type": "uint8",
+  "chunk_grid": {
+    "name": "regular",
+    "configuration": {
+      "chunk_shape": [
+        1
+      ]
+    }
+  },
+  "chunk_key_encoding": {
+    "name": "default",
+    "configuration": {
+      "separator": "/"
+    }
+  },
+  "fill_value": 0,
+  "codecs": [
+    {
+      "name": "bytes",
+      "configuration": {
+        "endian": "little"
+      }
+    }
+  ],
+  "attributes": {},
+  "storage_transformers": [
```

Comment:
Hmm. Since zarr-java doesn't know how to deal with this storage transformer, it should fail when trying to parse it. Right now, only empty array or missing "storage_transformers" key are acceptable.